package.json

```diff
@@ -358,7 +358,7 @@
     "semver": "^7.3.5",
     "serve-handler": "^6.1.2",
     "sinon": "^9.0.0",
-    "source-map": "^0.7.2",
+    "source-map": "^0.6.1",
```

Comment:
Why was this downgraded?

Reply:
no

Reply:
weird - source-map was last changed 4 years ago, not sure how this got in here